app/forms/idv/doc_pii_form.rb

```diff
@@ -58,8 +58,30 @@ def self.pii_like_keypaths
       keypaths
     end
 
+    # Modifies the errors object and consolidate/mutate it,
+    # used in image_upload_response_presenter to customizer
+    # error message rendering for pii errors
+    #
+    # errors: The DocPiiForm errors object
+    def self.present_error(existing_errors)
+      return if existing_errors.blank?
+      if existing_errors.any? do |k, v|
```

Comment:
Is this really needed? or we just trigger this if there is any error, since technically when we at there, it IS pii error.

Reply:
> or we just trigger this if there is any error, since technically when we at there, it IS pii error

Could there be a scenario where `existing_errors` includes non-pii errors?  If yes, I think we need to keep this check.  If no, then you probably don't need lines 67-69.

Nit:  I personally think lines 67-69 would be a little more readable as a one-liner.
`if existing_errors.any? { |k, v| PII_ERROR_KEYS.include?(k) }`

Reply:
@eileen-nava, the errors can be tricky, I will just leave it here just in case.